src/pages/APIs/PredictionAPI.ts

```diff
@@ -61,16 +61,37 @@ export const postSpeechToTextEndpoint = async ({
  */
 export const postTranslationEndpoint = async ({
   text,
+  languagePair,
 }: {
   text: string,
+  languagePair: { from: string, to: string },
```

Comment:
We can make this more type-safe by specifying that we only expect values that exist in `LanguageEnum` and not just a string

```suggestion
  languagePair: { from: LanguageEnum, to: LanguageEnum },
```